### ISSUE TITLE
facet rotate BROKEN

### ISSUE THREAD

**Issue by agrignard** [gama-platform/gama#2107]
### Steps to reproduce
1. We use to have a facet rotate for a display (introduced after the 1.7 official release)
e.g display city_display  type:opengl rotate:9.74 

### Expected behavior
Actually do the rotation

### Actual behavior
No effect at all...

### System and version 
Which version of GAMA, JDK, OS ... (if necessary, copy Help>Installation Details>Configuration in GAMA)

**Comment by agrignard** [comment]
In Abstract3DRenderer thhe value is read in 
```
commonInit(final GLAutoDrawable drawable)
currentZRotation = data.getZRotation();
```

but I don't think the rotation is done 

Should we add the rotation (a simple glRotate) just after that?

**Comment by AlexisDrogoul** [comment]
I'm on it. It's solved in my implementation. For a reason I ignore, setting the rotate parameter did not enable the rotation in the LayeredDisplayData. And the implementation of rotate model was strange, to say the least (no criticism, maybe I did it myself !).

**Comment by agrignard** [comment]
Any update on this? Is it possible to commit your change?

**Comment by AlexisDrogoul** [comment]
I will commit them soon. My concern is that some things are broken on the shaders part (but the good news is that the keystone is now implemented in the regular pipeline, as well as camera management)

**Comment by agrignard** [comment]
that's a great news (keystoning was not usable for long term demo because of memory consumption so that's great if it works with the normal pipeline) and It was the only purpose why I used the shader pipeline (which is I think to hard to maintain as Julien is not working on it). 

I am looking forward to test this (and to see how you mange to implement it in the regular pipeline, the shader version was tricky to understand)

**Comment by AlexisDrogoul** [comment]
Fixed in the latest commit (f775bfcde7b2167b4d7fc69e67b677638afd5592)
